src/test/unit/profile-data.test.js

```diff
@@ -130,14 +130,14 @@ describe('process-profile', function () {
       expect(profile.threads.length).toEqual(3);
     });
 
-    it('should not have a profile-wide libs property', function () {
```

Comment:
I agree it's a bit strange. No, I think it has always been this way.
Maybe the test was written because this is one of the most obvious ways in which the gecko format and the processed format differed.